crates/timer-interface-types/src/lib.rs

```diff
@@ -0,0 +1,27 @@
+#![no_std]
+
+use zerocopy::{AsBytes, FromBytes};
+use num_enum::{IntoPrimitive, TryFromPrimitive};
+
```

Comment:
Do you think it would be reasonable to add a struct that implements [`embedded_hal::timer::CountDown`](https://docs.rs/embedded-hal/latest/embedded_hal/timer/trait.CountDown.html) and uses this interface?

Reply:
Yes, as well as a periodic timer.